src/Shipping_Method/Settings.php

```diff
@@ -1191,4 +1213,17 @@ public function get_enable_logging() {
 	public function is_logging_enabled() {
 		return ( 'yes' === $this->get_enable_logging() );
 	}
+
+    /**
+     * Get all shipping options.
+     *
+     * @return array
+     */
+    public function get_default_shipping_options() {
+        $settings = get_option( 'woocommerce_postnl_settings' );
+        $default_shipping_options = isset( $settings['default_shipping_options'] ) ? $settings['default_shipping_options'] : array();
```

Comment:
Use the pre built function to get the option

```suggestion
return $this->get_country_option( 'default_shipping_options' );
```